### ISSUE TITLE
Optimize the GUI opening speed

### ISSUE THREAD

**Issue by dixiatielu** [Chew/TransmuteIt#31]
It took about 8 seconds for a player to open the GUI after performed the `/tm` command. The server also sent a warning of overloaded. It is necessary to optimize the GUI opening speed.
![image](https://user-images.githubusercontent.com/29675002/90305553-ae8cb700-def6-11ea-9dc6-a96f7d09d793.png)

**Comment by Chew** [comment]
What version is this on?

**Comment by dixiatielu** [comment]
Minecraft 1.15.2, paper build 385.

**Comment by Chew** [comment]
No no, the TransmuteIt version, that is useful too

**Comment by dixiatielu** [comment]
The latest developing version, downloaded from your jenkins server yesterday.

**Comment by Chew** [comment]
Ah alright, I'll look into that a bit.

**Comment by dixiatielu** [comment]
OK, thanks~

**Comment by Chew** [comment]
I couldn't reproduce this with your PR (as that's all I got around to testing atm), so either your PR fixed it or it might be an issue outside of the plugin's scope.

My recommendation (since you have Paper):
Run a `/timings start`
Do some non-laggy things
Run the GUI commands
Run `/timings report`

**Comment by dixiatielu** [comment]
I built my pr and tested it, the GUI still opens slowly with my pr. And [that's my timings report](https://timings.aikar.co/?id=c0ddc90b880c45ce8a50673837e0f7f0).

**Comment by dixiatielu** [comment]
I don't know why, I tested it in my real server, it didn't lag at all. But when I tested in my local server, it is still laggy.

**Comment by Chew** [comment]
Weird, do any other plugins with GUIs have similar effects?

**Comment by dixiatielu** [comment]
No. I use ChestCommands and GUIShop. Neither of them is laggy when opening the GUI.
